src/API.js

```diff
@@ -0,0 +1,59 @@
+const likeURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/lCNzVmZgTty5Cce1TAf3/likes/';
+
+async function fetchMeals(categoryName) {
```

Comment:
- [ ] It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

Reply:
Done